### ISSUE TITLE
Could you please let me know how many GPUs were used in the training process?

### ISSUE THREAD

**Issue by sqs-ustc** [qihao067/CrossFlow#5]

**Comment by qihao067** [comment]
Hello, 

The requirements mainly depend on the model size and image resolution. In my experience, 8 A100/H100 GPUs are sufficient to reproduce all the results. I used 8 GPUs for smaller models and 16 GPUs for the largest one. But I think you can train all models on 8 GPUs with gradient accumulation.

However, increasing the number of GPUs can significantly accelerate the training process. For example, using 32 or 64 GPUs to support a larger batch size (e.g., 4096 instead of 1024) will result in a much better model within the same number of iterations.

Thanks
